LaSalle TJ, Gonye ALK, Freeman SS, et al. **Longitudinal characterization of circulating neutrophils uncovers phenotypes associated with severity in hospitalized COVID-19 patients**. 
Cell Rep Med. 2022;3(10):100779. doi:10.1016/j.xcrm.2022.100779 IF: 11.7 Q1 
>- raw data is not provided. However, the read count matrix and TPM matrix used in this study are available in GEO under accession number GSE212041 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE212041).
All code required to run the analyses in this manuscript is deposited in Zenodo (https://doi.org/10.5281/zenodo.7030528) based on the associated Github repository (https://github.com/lasalletj/COVID_Neutrophils).

Duijvelaar E, Gisby J, Peters JE, Bogaard HJ, Aman J. Longitudinal plasma proteomics reveals biomarkers of alveolar-capillary barrier disruption in critically ill COVID-19 patients. Nat Commun. 2024;15(1):744. Published 2024 Jan 25. doi:10.1038/s41467-024-44986-w IF: 14.7 Q1
>- Pseudonymized source data utilised in this study and generated data are publicly available without restriction. The raw data required for deconvolution of RNA-seq analysis in human lung tissue can be obtained from DOI: 10.1038/s41586-021-03570-8IF: 50.5 Q1 .
>-For this study, external RNA-seq data from SARS-CoV-2 infected hamsters was retrieved from 10.1038/s41423-023-00985-3IF: 21.8 Q1 . SomaScan proteomics data from ref. can be accessed from 10.1016/j.xcrm.2021.100287IF: 11.7 Q1 . SomaScan proteomics data from ref. can be accessed from 10.1038/s41467-022-35454-4IF: 14.7 Q1 .
>- The R script required to reproduce the analyses is available from Zenodo (10.5281/zenodo.10142777).


Ma R, Sun ED, Donoho D, Zou J. Principled and interpretable alignability testing and integration of single-cell data. Proc Natl Acad Sci U S A. 2024;121(10):e2313719121. doi:10.1073/pnas.2313719121IF: 9.4 Q1
>- The human pancreatic data can be accessed in the R package SeuratData (https://github.com/satijalab/seurat-data) under the dataset name panc8. The PBMC data can be accessed in the R package SeuratData (https://github.com/satijalab/seurat-data) under the dataset name pbmcsca. The mouse brain chromatin accessibility data were downloaded from Figshare (https://figshare.com/ndownloader/files/25721789), containing a dataset from Fang et al. (65) (single-nucleus ATAC-seq protocol), and a 10X Genomics dataset for fresh adult mouse brain cortex (sample retrieved from https://support.10xgenomics.com/single-cell-atac/datasets/1.2.0/atac_v1_adult_brain_fresh_5k). Both ATAC-seq datasets have been preprocessed by Luecken et al. (6) to characterize gene activities.
>- The human lung data were downloaded as Anndata objects (samples 1, A3, B3 and B4) on Figshare (https://figshare.com/ndownloader/files/24539942). The human liver, MLN, and LLN immune cell data were downloaded from https://www.tissueimmunecellatlas.org. The mouse PBMC datasets (samples “Control 1h” and “LPS 1h”) were downloaded from Gene Expression Omnibus (GSE178431) https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178429. The mouse gastrulation seqFISH data were downloaded from https://content.cruk.cam.ac.uk/jmlab/SpatialMouseAtlas2020/, and the RNA-seq (10X Chromium) data can be accessed as “Sample 21” in MouseGastrulationData within the R package MouseGastrulationData.
>-  For the mouse VISP data, the ISS spatial transcriptomic data can be downloaded from https://github.com/spacetx-spacejam/data, the ExSeq spatial transcriptomic data can be downloaded from https://github.com/spacetx-spacejam/data, and the Smart-seq data can be downloaded from https://portal.brain-map.org/atlases-and-data/rnaseq/mouse-v1-and-alm-smart-seq.
>-  The R and Python packages of SMAI, and the R codes for reproducing our simulations and data analyses, are available at our GitHub repository https://github.com/rongstat/SMAI.

Drury RE, Camara S, Chelysheva I, et al. Multi-omics analysis reveals COVID-19 vaccine induced attenuation of inflammatory responses during breakthrough disease. Nat Commun. 2024;15(1):3402. Published 2024 Apr 22. doi:10.1038/s41467-024-47463-6
>- The gene expression data, such as 3rd gen long-read RNA-seq, next-gen short-read RNA-seq and small RNA-seq, generated in this study have been deposited in the Gene Expression Omnibus database under accession code GSE228842. The DGE data generated in this study are provided in the Supplementary Information. The Homo sapiens genome build GRCh38 used in this study is available in to download through HISAT2 website (https://genome-idx.s3.amazonaws.com/hisat/grch38_genome.tar.gz). Gene lists for the pathway analysis used in this study are available through Molecular Signatures Database (http://www.broadinstitute.org/gsea/msigdb/index.jsp). MicroRNA databases used in this study are available through miRNet (https://www.mirnet.ca). Source data are provided with this paper.
>- The reproducible code for analysis is available at Zenodo (10.5281/zenodo.10797098)89 and on Github https://github.com/Chelysheva/COVID_multiomics_codes/ and https://github.com/dan-scholar/COVID_RNAseq_script/.

Gygi JP, Maguire C, Patel RK, et al. Integrated longitudinal multiomics study identifies immune programs associated with acute COVID-19 severity and mortality. J Clin Invest. 2024;134(9):e176640. Published 2024 May 1. doi:10.1172/JCI176640
>- Data files are available at ImmPort Shared Data under accession number SDY1760 and in the NLM’s Database of Genotypes and Phenotypes (dbGaP) under accession number phs002686.v1.p1. All impacc-public-code analysis codes have been deposited in Bitbucket (https://bitbucket.org/kleinstein/impacc-public-code) and are publicly available. DOIs are included in Supplemental Table 11, which lists the key resources. Data required to re-create both the main and supplemental figures are included in the Supporting Data Values file.

Viode A, Smolen KK, van Zalm P, et al. Longitudinal plasma proteomic analysis of 1117 hospitalized patients with COVID-19 identifies features associated with severity and outcomes. Sci Adv. 2024;10(21):eadl5762. doi:10.1126/sciadv.adl5762
>-Data and materials availability: All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. Data files are available at ImmPort (immport.org) under accession number SDY1760. The code used for data analysis and visualization is available at Github page: https://github.com/SteenOmicsLab/COVID19 and deposited to Zenodo: https://zenodo.org/records/10913186.

Yang Y, Dong L, Li Y, Huang Y, Zeng X. Summary data-based Mendelian randomization and single-cell RNA sequencing analyses identify immune associations with low-level LGALS9 in sepsis. J Cell Mol Med. 2024;28(14):e18559. doi:10.1111/jcmm.18559
>- The analysis code to produce the major results presented in the paper can be accessed from https://github.com/yangyongsan‐bio/Sepsis‐SMR‐scRNA.


































